R/main.R

```diff
@@ -34,14 +34,19 @@ Options:
 
 main_worker_args <- function(args = commandArgs(TRUE)) {
   usage <- "Usage:
-orderly_worker [<queue_id>]"
+orderly_worker [options] [<queue_id>]
+
+Options:
+  --go-signal=PATH  Absolute path for go signal"
   dat <- docopt_parse(usage, args)
-  list(queue_id = dat$queue_id)
+  list(queue_id = dat[["queue_id"]],
+       go_signal = dat[["go_signal"]])
 }
 
 main_worker <- function(args = commandArgs(TRUE)) {
   # nocov start
   args <- main_worker_args(args)
+  wait_for_go_signal(path, args$go_signal)
```

Comment:
If --go-signal is not supplied we start immediately, right?

Reply:
Yeah that's right